docs/install_guide.adoc

```diff
@@ -0,0 +1,140 @@
+| aa4363e8f74641ce9a53c14fd4ce8184 | regionOne | placement    | placement    | True    | public    | https://placement-public-openstack.apps-crc.testing      |
+| b313c0ce41b840a884b8e0ac4a5a819f | regionOne | cinderv3     | volumev3     | True    | public    | https://cinder-public-openstack.apps-crc.testing/v3      |
+| ccadeae4322346fe9feee6bb0677b014 | regionOne | nova         | compute      | True    | internal  | http://nova-internal.openstack.svc:8774/v2.1             |
+| dfac9cb0706d411b8c90c6df264fb575 | regionOne | nova         | compute      | True    | public    | https://nova-public-openstack.apps-crc.testing/v2.1      |
+| e8314601b5184f74b99b0815e1d57d5e | regionOne | cinderv3     | volumev3     | True    | internal  | http://cinder-internal.openstack.svc:8776/v3             |
++----------------------------------+-----------+--------------+--------------+---------+-----------+----------------------------------------------------------+
+----
+
+Now verify that Telemetry operator with Prometheus metric storage is ready:
+
+[,bash]
+----
+$ oc get telemetry
+NAME        STATUS   MESSAGE
+telemetry   True     Setup complete
+
+$ oc get route
+NAME                           HOST/PORT                                                 PATH   SERVICES                       PORT                           TERMINATION     WILDCARD
+cinder-public                  cinder-public-openstack.apps-crc.testing                         cinder-public                  cinder-public                  edge/Redirect   None
+glance-default-public          glance-default-public-openstack.apps-crc.testing                 glance-default-public          glance-default-public          edge/Redirect   None
+horizon                        horizon-openstack.apps-crc.testing                               horizon                        horizon                        edge/Redirect   None
+keystone-public                keystone-public-openstack.apps-crc.testing                       keystone-public                keystone-public                edge/Redirect   None
+metric-storage-alertmanager    metric-storage-alertmanager-openstack.apps-crc.testing           metric-storage-alertmanager    web                                            None
+metric-storage-prometheus      metric-storage-prometheus-openstack.apps-crc.testing             metric-storage-prometheus      web                            edge/Redirect   None
+neutron-public                 neutron-public-openstack.apps-crc.testing                        neutron-public                 neutron-public                 edge/Redirect   None
+nova-novncproxy-cell1-public   nova-novncproxy-cell1-public-openstack.apps-crc.testing          nova-novncproxy-cell1-public   nova-novncproxy-cell1-public   edge/Redirect   None
+----
+
+Prometheus metrics will be viewable in a web browser at https://metric-storage-prometheus-openstack.apps-crc.testing.
```

Comment:
Prometheus metrics will be viewable in a web browser at the showed HOST/PORT, i.e.: https://metric-storage-prometheus-openstack.apps-crc.testing